src/main/resources/import.sql

```diff
@@ -163,4 +187,9 @@ INSERT INTO value (value, description, header_id, category_id) VALUES ("*", "Nev
 # =========== X-Runtime ===========
 INSERT INTO value (value, description, header_id, category_id) VALUES ("*", "Never use this header in a live situation this makes the server an easier target.", 15, 2);
 
+# =========== X-Content-Security-Policy ===========
+INSERT INTO value (value, description, header_id, category_id) VALUES ("*", "It's recomended not to use this header because it is outdated, use 'Content-Security-Policy' instead. Note: Having both 'Content-Security-Policy' and 'X-Content-Security-Policy' active and the same can cause problems with diffrent browsers. ", 16, 3);
```

Comment:
"It is not recommended to use this header because...", 
"and the same" -> "at the same time"??